go/client/cmd/ctclient.go

```diff
@@ -0,0 +1,108 @@
+package main
```

Comment:
You probably want to put this in a `ctclient/` directory (I'm assuming you want the binary to be called `ctclient` from the examples you provided)